dashboard/app/views/plc/learning_modules/new_learning_resource_for_module.html.haml

```diff
@@ -0,0 +1,37 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h3{style: 'margin-bottom: 24px;'}
+  Add a resource (Module: #{@learning_module.name})
+
+.new_resource_section{style: 'float: left; width: 33%'}
+  %p
+    = link_to 'New Script', new_script_path, class: 'btn'
```

Comment:
They aren't.

This is what you click if you want to build a new script or level. If you want to create associate a learning module with a given script, create a script completion task, which is associated with both a script and a learning module.